docs/_packages/socket_mode.md

```diff
@@ -0,0 +1,238 @@
+---
+title: Socket Mode
+permalink: /socket-mode
+order: 7
+anchor_links_header: Usage
+---
+
+# Slack Socket Mode
+
+
+## Installation
+
+```shell
+$ npm install @slack/socket-mode
+```
+
+---
+
+### Initialize the client
+
+The package is designed for the newly released [**Socket Mode**](https://api.slack.com/socket-mode) feature. It exports a `SocketModeClient` class. Your app will create an instance of the class for each workspace it
+communicates with. Creating an instance requires an **app level token** from Slack. Apps connect to the **Socket Mode** API using the newly introduced **app level token**, which starts with `xapp`.
+
+Note: **Socket Mode** requires the scope `connections:write`. In your browser, navigate to your [app's app config](https://api.slack.com/apps) and go to the **OAuth and Permissions** section to add the scope.
+
+
+```javascript
+const { SocketModeClient } = require('@slack/socket-mode');
+
+// Read a token from the environment variables
+const appToken = process.env.SLACK_APP_TOKEN;
+
+// Initialize
+const client = new SocketModeClient({appToken});
+```
+
+### Connect to Slack
+
+Data from Slack will begin to flow to your app once the client is connected. You'll also be able to send data to
```

Comment:
```suggestion
After your client establishes a connection, your app can send data to and receive data from Slack.
```

Reply:
Weird thing is happening with suggestions, but if you take suggestion above, remove the fragment below (but keep the "Connecting..." sentence)